oat-sa/extension-tao-testcenter
# feat: rename Proctor/Un-Proctor buttons, update translations

Renamed buttons on eligibility management table:
_Proctor -> Require authorization
Un-Proctor -> Monitor only_
 
Related to : https://oat-sa.atlassian.net/browse/TR-2750
  
#### How to test
 
 Throwaway environment linked in the related ticket

- Create a test center
- Add Eligible Delivery
- Verify buttons have been renamed

![Screenshot from 2021-12-07 17-08-15](https://user-images.githubusercontent.com/2705327/145065588-021a9cb5-38a2-4998-ada1-c39060b30ac2.png)
